config/database.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/kelseyhightower/envconfig"
+	_ "github.com/kelseyhightower/envconfig"
+	"gopkg.in/yaml.v2"
+	"os"
+)
+
+var DB *sql.DB
+
+// Config represents configuration
+type Config struct {
+	Database struct{
+		Host     string `yaml:"host", envconfig:"DB_HOST"`
```

Comment:
run gofmt.

Reply:
Can we create another struct for Database? You can better expand this for other configurations then.

Reply:
done